Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN mkdir -p /usr/share/man/man1 && \
 COPY requirements.txt setup.py /app/
 WORKDIR /app
 RUN pip install --user -r requirements.txt --no-warn-script-location && \
-    pip install --user entmax && \
+    pip install --user entmax stanfordnlp && \
```

Comment:
Perhaps better moved to `setup.py`, like other dependencies. IIRC, `entmax` was special because it hardcoded PyTorch dependency in a weird way, so we had to control the order of its installation.

Reply:
I corrected as you commented.